config/settings/base.py

```diff
@@ -26,7 +26,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ["*"]
```

Comment:
Depois isto convém limitar ao dominio em que for feito deploy das aplicações. Assim há segurança que os pedidos externos de aplicações não autorizadas não passem. Em produção é má pratica ter *

Reply:
hm então neste momento devia por só o dev.localhost ?